examples/README.txt

```diff
@@ -0,0 +1,10 @@
+***************
+Example Gallery
+***************
+
+This gallery contains examples of how to use sunkit-pyvista.
+Most of these examples require the sunpy sample data, which you can download by running::
+
+     >>> import sunpy.data.sample
+
+Once downloaded the data will be stored in your user directory and you will not need to download it again.
```

Comment:
I am not sure if this will be true going forward but the first example does the sample data.

Reply:
We leave it in for now I guess?